runtime/seccomp_filter.c

```diff
@@ -79,6 +70,6 @@ int configure_seccomp(void) {
     perror("seccomp(SECCOMP_FILTER_FLAG_TSYNC)");
     return -1;
   }
-  // prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, &prog);
```

Comment:
This was debugging cruft; this form is deprecated and strictly less flexible compared to the standalone `seccomp()` syscall.